src/dsp_tools/fast_xmlupload/upload_files.py

```diff
@@ -385,6 +387,9 @@ def upload_files(
     """
     dir_with_processed_files_path = _check_processed_dir(dir_with_processed_files)
     pkl_file_paths = get_pkl_files()
+    logger.info("Found the following pickle files:")
+    for pkl in pkl_file_paths:
+        logger.info(f" - {str(pkl)}")
 
     # read paths from pkl file
     internal_filenames_of_processed_files = _get_paths_from_pkl_files(pkl_files=pkl_file_paths)
```

Comment:
I'm not really familiar with this code - I'm assuming the paths will be logged too, somewhere?

Reply:
Yes, every time a file is uploaded successfully, its path is logged.